stl/inc/__msvc_string_view.hpp

```diff
@@ -221,7 +221,15 @@ struct _WChar_traits : private _Char_traits<_Elem, unsigned short> {
         // compare [_First1, _First1 + _Count) with [_First2, ...)
 #if _HAS_CXX17
         if constexpr (is_same_v<_Elem, wchar_t>) {
+#if _HAS_CXX20
+            if (_STD _Is_constant_evaluated()) {
+                return __builtin_wmemcmp(_First1, _First2, _Count);

```

Comment:
It seems that we should say `// TRANSITION, GH-2289` or something like.

Does Clang's `__builtin_wmemcmp` behave fine? If so, we should probably unconditionally use `__builtin_wmemcmp` for Clang.

Reply:
Clang simply generates a call to wmemcmp/wmemchr, see https://godbolt.org/z/qM9zs77Pj and https://godbolt.org/z/j5c69chsc.